electrum/gui/qt/main_window.py

```diff
@@ -2895,7 +2895,8 @@ def sign_verify_message(self, address=''):
         layout.addWidget(QLabel(_('Address')), 2, 0)
         layout.addWidget(address_e, 2, 1)
 
-        signature_e = QTextEdit()
+        signature_e = ShowQRTextEdit(config=self.config)
+        signature_e.setReadOnly(False)
         signature_e.setAcceptRichText(False)
```

Comment:
This line is no longer needed, as `ShowQRTextEdit` inherits `QPlainTextEdit`, so it is incapable of rich text, and this method is undefined.

```
Traceback (most recent call last):
  File "/home/user/wspace/electrum/electrum/gui/qt/address_list.py", line 270, in <lambda>
    menu.addAction(_("Sign/verify message"), lambda: self.parent.sign_verify_message(addr))
  File "/home/user/wspace/electrum/electrum/gui/qt/main_window.py", line 2900, in sign_verify_message
    signature_e.setAcceptRichText(False)
AttributeError: 'ShowQRTextEdit' object has no attribute 'setAcceptRichText'
```

Reply:
>  Could you suggest some test to perform?

I just meant try signing and verifying a message, show a qr code -- just play around with the functionality that is being changed :)
To catch issues like the one above.

Reply:
Ok, I will try. Thank you!